scripts/scripts.js

```diff
@@ -69,7 +69,10 @@ export function toClassName(name) {
  */
 function wrapSections($sections) {
   $sections.forEach(($div) => {
-    if (!$div.id) {
+    if (!$div.children.length) {
```

Comment:
It is probably valid because all text should be wrapped inside `<p>` tags but according to the doc:`Element.children includes only element nodes. To get all child nodes, including non-element nodes like text and comment nodes, use Node.childNodes.`. https://developer.mozilla.org/en-US/docs/Web/API/Element/children
You might remove a div with a text node.

Reply:
You're right: I should use `childNodes` instead.